app/helpers/facets_helper.rb

```diff
@@ -55,4 +55,9 @@ def pivot_facet_field_in_params?(pivot)
     pivot.each { |field| in_params = true if params[:f] && params[:f][field] }
     in_params
   end
+
+  def facet_item_component_class(facet_config)
+    default_component = facet_config.pivot ? Blacklight::FacetItemPivotComponent : PsulFacetItemComponent
```

Comment:
@awead - By overriding this method, we can user our custom component and subclass the default component, avoiding all that code duplication of my previous approach.